docs/commands/ts.add.md

```diff
@@ -94,22 +99,18 @@ The complexity of `TS.ADD` is always `O(M)`, where `M` is the number of compacti
 
 <details open><summary><b>Append a sample to a temperature time series</b></summary>
 
-Create a temperature time series.
+Create a temperature time series and append a sample.
```

Comment:
Create a temperature time series, set its retention to 3.6 seconds, and append a sample.

- l suggest changing to a larger retention period - e.g., 1 year (31536000000), to make the example more realistic.

Add:  Note that if time series with such a name already exists - the sample would be added, but the retention won't be changed.